app/views/contexts/summary/question.py

```diff
@@ -125,14 +136,21 @@ def _concatenate_answers(self, answer_store, concatenation_type):
                 continue
 
             values_to_concatenate.extend(
-                answer_value if isinstance(answer_value, list) else [answer_value]
+                # Type ignore: This will be optional as we have answer_value check
+                answer_value
+                if isinstance(answer_value, list)
+                else [answer_value]  # type: ignore
             )
```

Comment:
Can be fixed by updating the code to something mypy can understand, i.e
```py
        values_to_concatenate: list[AnswerValueEscapedTypes] = []
        for answer_value in answer_values:
            if not answer_value:
                continue

            if isinstance(answer_value, list):
                values_to_concatenate.extend(answer_value)
            else:
                values_to_concatenate.append(answer_value)
```

Reply:
👍